Tmain/readtags-qualifier-op-downcase-upcase.d/stdout-expected.txt

```diff
@@ -0,0 +1,14 @@
+# (downcase $name)
+C	input.cpp	/^class C: A, B {$/
+# (member "a" (downcase $inherits))
+C	input.cpp	/^class C: A, B {$/
+# (member "b" (downcase $inherits))
+C	input.cpp	/^class C: A, B {$/
+# (member "c" (downcase $inherits))
+# (upcase $name)
+C	input.cpp	/^class C: A, B {$/
```

Comment:
C is already being "upcase'ed" in the output.tags. So this cannot test the behavior of "upcase" operator.

What your code tests:
```
(member "C" (upcase "C"))
```

B and C, the super classes of C are also already "upcase'ed'.
See output.tags. 
```
f	input.cpp	/^  int f (void) {$/;"	f	class:A	typeref:typename:int	file:
g	input.cpp	/^  int g (void) {$/;"	f	class:B	typeref:typename:int	file:
h	input.cpp	/^  int h (void) {$/;"	f	class:C	typeref:typename:int	file:
```
You can use f, g, and h for testing upcase operator like:

```
echo '# (upcase $name)'
${V} ${READTAGS} -t output.tags -Q '(eq? "F" (upcase $name))' -l
```